modules/docs/mdx/8.0-UPGRADE-GUIDE.mdx

```diff
@@ -357,15 +457,21 @@ return (
 );
 ```
 
-#### Overflow behavior
+Read more about [moving from Router DOM v5 to v6](https://reactrouter.com/en/main/upgrading/v5).
```

Comment:
This felt a little out of scope for the Upgrade Guide, but I could be convinced otherwise. I've left it in for now.

Reply:
Yeah, we could remove "Read more about [moving from Router DOM v5 to v6](https://reactrouter.com/en/main/upgrading/v5)."